vendor/ocaml-blake3-mini/blake3_dispatch.c

```diff
@@ -5,7 +5,7 @@
 #include "blake3_impl.h"
 
 #if defined(_MSC_VER)
-#include <windows.h>
+#include <Windows.h>
```

Comment:
is this change intentional?

Reply:
Yeah, let's move out this change out of here. #11940 should have introduced a patch file